example/hibernate-javax/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
+    alias(libs.plugins.kotlin.noarg)
+    alias(libs.plugins.kotlin.allopen)
+    alias(libs.plugins.kotlin.jpa)
+}
+
+dependencies {
+    @Suppress("VulnerableLibrariesLocal", "RedundantSuppression")
+    implementation(projects.example)
+    implementation(projects.jpqlDsl)
+    implementation(projects.jpqlRender)
+    implementation(libs.javax.hibernate.core)
+    implementation(projects.hibernateJavaxSupport)
+    implementation(libs.logback)
+
+    implementation(libs.test.h2)
+}
+
+kotlin {
+    jvmToolchain(8)
+}
+
+noArg {
+    annotation("com.linecorp.kotlinjdsl.example.jpql.hibernate.javax.annotation.CompositeId")
+    annotation("javax.persistence.Entity")
+    annotation("javax.persistence.Embeddable")
+}
+
+allOpen {
+    annotation("com.linecorp.kotlinjdsl.example.jpql.hibernate.javax.annotation.CompositeId")
+    annotation("javax.persistence.Entity")
+    annotation("javax.persistence.Embeddable")
+}
+
```

Comment:
```suggestion
kover {
     excludeInstrumentation {
         packages("org.hibernate.*")
     }
}
```
Adding this line seems to resolve the test report failure.
When kover performs a bytecode instrument to measure coverage, it conflicts with the bytecode instrument performed by hibernate.